qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -54,20 +55,25 @@ def circuits(self):
 
         sub_circuits = []
         sub_qubits = []
+        sub_maps = []
         sub_size = []
         num_qubits = 0
 
         # Generate data for combination
-        for expr in self._experiments:
+        for sub_exp in self._experiments:
+
+            # Generate transpiled subcircuits
+            circuits = sub_exp._transpiled_circuits()
+
             # Add subcircuits
-            circs = expr.circuits()
-            sub_circuits.append(circs)
-            sub_size.append(len(circs))
+            sub_circuits.append(circuits)
+            sub_size.append(len(circuits))
 
             # Add sub qubits
-            qubits = list(range(num_qubits, num_qubits + expr.num_qubits))
+            qubits = list(range(num_qubits, num_qubits + sub_exp.num_qubits))
```

Comment:
Why we cannot simply count qubits using `set()` of physical qubits provided by instances?

Reply:
The order matters

Reply:
The relative order of sub experiments for the logical circuits matters.